Sources/WendyAgent/Services/OCI+Entitlements.swift

```diff
@@ -93,30 +93,33 @@ extension OCI {
                 case .host:
                     // Remove any network namespace to ensure host networking
                     self.linux.namespaces.removeAll(where: { $0.type == "network" })
-                    // Bind mount host's /etc/resolv.conf for DNS resolution
+
+                    // Mount systemd-resolved's actual resolv.conf for DNS resolution
+                    // We use /run/systemd/resolve/resolv.conf instead of /etc/resolv.conf
+                    // because /etc/resolv.conf often points to 127.0.0.53 (systemd-resolved stub)
+                    // which doesn't work in containers. The /run path contains the real upstream DNS servers.
```

Comment:
It did work in my container actually. Shouldn't we mount _both_? I'd imagine some DNS clients that run in-process only look at `/etc/resolv.conf`.